_projects/beer.md

```diff
@@ -12,19 +12,25 @@ box: >
 
 ---
 
-## Technology
+# 2020/2021 Recruitment
 
-### Chemistry
+We are now accepting applications for all sub-teams from August 25th to September 14th 2020. You can check out our recruitment page [here](https://www.ubcenvision.com/recruitment/#BIOT).
+
+If you have any question about recruiting or want to talk about beer, feel free to contact biot@ubcenvision.com.
+
+# Technology
+
+## Chemistry
 Fermentation is an incredibly important biological process. It is used in the production of penicillin, wine, kombucha, and of course, beer! Yeast is a facultative anaerobe meaning that it can two different metabolic pathways. The aerobic pathway uses oxygen to produce water and carbon dioxide. The anaerobic pathway uses sugars broken down from the mashing process to produce ethanol and carbon dioxide. Hops is added for flavor. Each strain of hops has its own characteristics such as alpha acids and flavor profile. A mixture of hops makes up the flavor in each distinct beer type. This team researches the reaction kinetics in the fermenter as well as makes the recipes for the beer (and soon kombucha) we brew! This team is also responsible for testing key characteristics of the beer such as bitterness, alcohol by volume, turbidity, and the standard reference model (or color) of the beer. 
 
-### Software
+## Software
 In a large factory everything in controlled in a central control room so why not apply the same for a home brewery? This team takes the data that is read from the temperature sensors and sends it to Amazon Web Services where the data is processed and sent to a web app viewable on desktop or mobile. This data can be put into a graph to track temperature profiles or the acidity can be used predict the final SRM. 
 
-### Instrumentation
+## Instrumentation
 Custom instrumentation is built to read temperature and acidity. We are developing a way to read the SRM (or color) of the beer using a Raspberry Pi camera. Taking the input variables allows us to use them in machine learning algorithms later custom built for our brewing system. Many companies are entirely built around building and designing custom brewing instrumentation. A refrigeration system is used to chill the beer to desired temperatures in a food safe manner. Our next step is using the temperature readings to control the refrigeration system. 
 
-### Machine Learning
+## Machine Learning
 Process data is incredibly difficult to analyze and predict. There are many unpredictable variables that can only be analyzed in a computer. Running machine learning algorithms to predict output variables will allow us to predict and create beer tailored to whatever the brewer desires. We are currently partnering with a local brewery to analyze key process variables for our machine learning algorithms. 
 
-## Like Brewing? 
+# Like Brewing? 
 If you have any questions or want to talk about beer feel free to contact brewing@ubcenvision.com
```

Comment:
Above you use a biot@ubcenvision.com email - maybe this should be changed to it as well so that it is consistent